### PR TITLE
Wd 657 migrate serial ingress from v 0 to v 1

### PR DESCRIPTION
**Pull request by GhasBarbouchi**
update the module tag

### CODE REVIEW

File: Module.yaml [hunk 1]
```diff
@@ -57,6 +57,8 @@ envs:                                   # Array of environment variables that wi
      - 1
      - 1.5
      - 2
+ envDevices:
+      - PORT
```
Comment (ersanyamarya) [inline Module.yaml:61]: PORT is not a device.
Put the key on the env that has the device path

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+# Python specific
+env/
+__pycache__/
+
+# Macbook specific
+.DS_Store
+
+# vscode specific
+.vscode/*.log
+=======
```
Comment (ersanyamarya) [inline .gitignore:13]: Why do we have merge conflict indicator in the file?

File: .gitignore [hunk 3]
```diff
@@ -404,4 +417,5 @@ temp/
 .history
 .ionide
 
-# End of https://www.toptal.com/developers/gitignore/api/visualstudiocode,node,python,flask,macos,linux
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/visualstudiocode,node,python,flask,macos,linux
+>>>>>>> 88304b6079d7b2bac51c06fc9a74e22583baa5f8
```
Comment (ersanyamarya) [inline .gitignore:421]: Merge conflit indicators ??

**Review by ersanyamarya — CHANGES_REQUESTED** [review]
Please address the comments.

File: Module.yaml (hunk 1 above)
Comment (GhasBarbouchi) [reply Module.yaml:61]: Do you mean I have to put '/dev/ttyUSB0'?

**Review by ersanyamarya — APPROVED** [review]
LGTM

**Pull request merged**